services/contract_create.proto

```diff
@@ -82,12 +82,26 @@ import "duration.proto";
  *    If there are no admin keys, then it cannot be changed after the smart contract is created.
  */
 message ContractCreateTransactionBody {
+
     /**
-     * the file containing the smart contract byte code. A copy will be made and held by the
-     * contract instance, and have the same expiration time as the instance. The file is referenced
-     * one of two ways:
+     * There are two ways to specify the initcode of a ContractCreateTransction. If the initcode is 
+     * large (> 5K) then it must be stored in a file as hex encoded ascii. If it is small then it may 
+     * either be stored as a hex encoded file or as a binary encoded field as part of the transaciton.
```

Comment:
So just to confirm, the `initcode` is not hex encoded binary like the file data. It is the raw bytes?

Reply:
Yes, raw bytes.